build.gradle

```diff
@@ -8,9 +8,9 @@ buildscript {
     ]
 
     ext.deps = [
-            appCompat             : "com.android.support:appcompat-v7:25.0.0",
-            recyclerView          : "com.android.support:recyclerview-v7:25.0.0",
-            expandablerecyclerview: "com.github.thoughtbot.expandable-recycler-view:expandablerecyclerview:v1.3",
+            appCompat             : 'androidx.appcompat:appcompat:1.0.0',
+            recyclerView          : 'androidx.recyclerview:recyclerview:1.0.0',
+            expandablerecyclerview: 'com.github.deadman96385:expandable-recycler-view:v1.8',
```

Comment:
Regarding the expandablerecyclerview don't you think relying on your expandablerecyclerview implementation might be a little risky? If you removed it, there might be a problem. Perhaps consider a different option, such as copying the ARR into the library would be a good idea. What do you think?

Reply:
Yeah that's an option, I wasn't sure what you wanted for to do, so I matched the existing. If you want you could fork it and we point at that instead. Then you would have final say, and if it needs future updates I am more then willing to pull request there.

Reply:
Forked 👍🏼. Can you please update the implementation to use mine? Thank you.

```
implementation 'com.github.AliAsadi:expandable-recycler-view:v1.8'
```

Reply:
Should be good to go now, I validated my branch in android studio with building and testing the sample app.